### ISSUE TITLE
Create Implementation Guide

### ISSUE THREAD

**Issue by darrellodonnell** [trustoverip/tswg-trust-registry-protocol#4]
Create a placeholder for an implementation guide. Start moving content over to it:

- TRP Patterns commentary
- Data Model Diagram

**Comment by andorsk** [comment]
@darrellodonnell following up on this. Do we want to get the starting point on this? An implementation guide has been requested by a number of members but right now we are focused on the specification. 

We could easily start a folder in this repo for folks to work on this in parallel.

**Comment by andorsk** [comment]
Following up on this. We have references of Implementation Considerations here: https://trustoverip.github.io/tswg-trust-registry-protocol/#implementation-considerations and have left the rest of the Implementation Guide out of scope for the spec. 

Any issues with me closing this?
